Code/Archipelago.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Archipelago.MultiClient.Net;
+using Archipelago.MultiClient.Net.Enums;
+using Archipelago.MultiClient.Net.MessageLog.Messages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using ModCore;
+
+namespace ArchipelagoRandomizer
+{
+    public class Archipelago
+    {
+        public static ArchipelagoSession Session;
```

Comment:
Convention states upper camel casing is for properties, so I'd make that an auto property by adding
`{ get; private set; }` to it, or make it lowercase